app/src/main/java/com/fmpdroid/moneybox/dto/MoneyBoxDto.java

```diff
@@ -0,0 +1,57 @@
+package com.fmpdroid.moneybox.dto;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor @Data
+public class MoneyBoxDto implements Parcelable {
+
+    private String name;
+    private String description;
+    private float remainingAmount;
+    private float targetAmount;
+    private String dateCreated;
+    private String targetDate;
+    private boolean allowReminder;
+
+    protected MoneyBoxDto(Parcel in) {
+        name = in.readString();
+        description = in.readString();
+        remainingAmount = in.readFloat();
+        targetAmount = in.readFloat();
+        dateCreated = in.readString();
+        targetDate = in.readString();
+        allowReminder = in.readByte() != 0;
+    }
+
+    public static final Creator<MoneyBoxDto> CREATOR = new Creator<MoneyBoxDto>() {
+        @Override
+        public MoneyBoxDto createFromParcel(Parcel in) {
+            return new MoneyBoxDto(in);
+        }
+
+        @Override
+        public MoneyBoxDto[] newArray(int size) {
+            return new MoneyBoxDto[size];
+        }
+    };
```

Comment:
Unwanted code. it has not purpose/usage

Reply:
these are required for parcelable thingy